vllm/model_executor/layers/linear.py

```diff
@@ -753,18 +753,23 @@ def forward(self, input_):
 
         # Matrix multiply.
         assert self.quant_method is not None
-        output_parallel = self.quant_method.apply(self, input_parallel)
+        bias_ = None if (self.tp_size > 1 or self.skip_bias_add) else self.bias
```

Comment:
Could you add a comment explaining what happens when tp_size > 1? I think it should say something like:
```
# Fuse bias into the GEMM in the TP == 1 case.
# The TP > 1 case is problematic as the bias will be redundantly summed during the AllReduce if fused
```

Reply:
Added a comment (although it reads a bit differently since the TP>1 case is now handled).